scripts/ci-e2e.sh

```diff
@@ -52,13 +78,41 @@ source "${REPO_ROOT}/hack/ensure-kustomize.sh"
 # shellcheck disable=SC1091,SC1090
 source "${M3_DEV_ENV_PATH}/lib/images.sh"
 # shellcheck disable=SC1091,SC1090
-source "${M3_DEV_ENV_PATH}/lib/releases.sh"
-# shellcheck disable=SC1091,SC1090
```

Comment:
After going through this change once again, I noticed that we are already sourcing other scripts here (images.sh, ironic related etc), what is the benefit of not sourcing release.sh the way it was and just set _FROM and _TO vars based on the already populated vars from m3-dev-env make run? That would avoid duplicate code and reduce maintenance of same scripts at 2 different locations.
In short, I am suggesting remove all these release fetching and setting CAPI/CAPM3 versions based on it, and rather  source releases.sh as before, and just set _FROM and _TO  based on the vars from `make` ,